tests/tape_tests/conftest.py

```diff
@@ -2,14 +2,198 @@
 import numpy as np
 import pandas as pd
 import dask.dataframe as dd
-
 import pytest
+import tape
+
 from dask.distributed import Client
 
 from tape import Ensemble
 from tape.utils import ColumnMapper
 
 
+def create_test_rows():
+
+    num_points = 1000
+    all_bands = np.array(["r", "g", "b", "i"])
+
+    rows = {
+        "id": 8000 + (np.arange(num_points) % 5),
+        "time": np.arange(num_points),
+        "flux": np.arange(num_points) % len(all_bands),
+        "band": np.repeat(all_bands, num_points / len(all_bands)),
+        "err": 0.1 * (np.arange(num_points) % 10),
+        "count": np.arange(num_points),
+        "something_else": np.full(num_points, None),
+    }
+
+    return rows
+
+def create_test_column_mapper():
+
+    return ColumnMapper(id_col="id", time_col="time", flux_col="flux", err_col="err", band_col="band")
+
+def create_test_source_table(npartitions=1):
+
+    return dd.from_dict(
+        create_test_rows(), 
+        npartitions)
+
+def create_test_object_table(npartitions=1):
+
+    n_obj = 5
+    id = 8000 + np.arange(n_obj)
+    name = id.astype(str)
+    return dd.from_dict(
+        dict(id=id, name=name),
+        npartitions)
+
+
+# pylint: disable=redefined-outer-name
+@pytest.fixture
```

Comment:
Do you expect to want to re-use these fixtures outside of `test_ensemble.py`? I usually think of the fixtures within the `conftest.py` as ones that you want to share extensively between different test modules. I think in this case, you can define the fixtures within `test_ensemble.py` directly, just to avoid bloating this shared file.

Reply:
New issue https://github.com/lincc-frameworks/tape/issues/257